application/build.gradle

```diff
@@ -25,14 +33,61 @@ jib {
     }
 }
 
+var databaseUrl = "jdbc:sqlite:${rootProject.buildDir}/database.db"
+
+flyway {
+    url = databaseUrl
+    locations = ["filesystem:src/main/resources/db"]
+}
+
+jooq {
+    version = "3.15.2"
+
+    configurations {
+        main {
+            generationTool {
+                jdbc {
+                    driver = 'org.sqlite.JDBC'
+                    url = databaseUrl
+                }
+                generator {
+                    name = 'org.jooq.codegen.DefaultGenerator'
+                    database {
+                        name = 'org.jooq.meta.sqlite.SQLiteDatabase'
+                        excludes = 'flyway_schema_history|sqlite_sequence'
+                    }
+                    generate {
+                        records = true
+                        immutablePojos = true
+                        fluentSetters = true
+                    }
+                    target {
+                        packageName = 'org.togetherjavabbot.tjbot.db.generated'
```

Comment:
Why `org.togetherjavabbot` while in other places its just `org.togetherjava`?

Reply:
Because I can't autocomplete, thanks :)